oem/ibm/libpldmresponder/utils.cpp

```diff
@@ -449,15 +449,16 @@ using namespace pldm::utils;
 int pldm::responder::oem_ibm_utils::Handler::setCoreCount(
     const EntityAssociations& Associations, const EntityMaps entityMaps)
 {
-    int coreCountRef = 0;
+    int coreCount = 0;
     // get the CPU pldm entities
     for (const auto& entries : Associations)
     {
         auto parent = pldm_entity_extract(entries[0]);
         // entries[0] would be the parent in the entity association map
         if (parent.entity_type == PLDM_ENTITY_PROC)
         {
-            int& coreCount = coreCountRef;
+            // Need to reset the count for each of the processors
+            coreCount = 0;
```

Comment:
just make this `int coreCount =0` ?

Reply:
We needed like this as we need to return the core count for unit test code.